src/neuroconv/datainterfaces/ecephys/phy/phydatainterface.py

```diff
@@ -12,7 +12,7 @@ class PhySortingInterface(BaseSortingExtractorInterface):
     def __init__(
         self,
         folder_path: FolderPathType,
-        exclude_cluster_groups: Optional[list] = None,
+        exclude_cluster_groups: Optional[list[str]] = None,
```

Comment:
?

Reply:
oh, whoops don't know how that got in there